admiral/pkg/clusters/types.go

```diff
@@ -124,6 +135,32 @@ func NewRemoteRegistry(ctx context.Context, params common.AdmiralParams) *Remote
 	}
 }
 
+type dependencyLookupCache struct {
+	sourceDestinations map[string][]string
```

Comment:
should we call this `destinationBySource`, or `sourceToDestinations`?

Reply:
`sourceToDestinations`